joy_teleop/package.xml

```diff
@@ -12,6 +12,7 @@
 
   <exec_depend>control_msgs</exec_depend>
   <exec_depend>rclpy</exec_depend>
+  <exec_depend>action_tutorials_interfaces</exec_depend>
```

Comment:
Hi @youtalk, thanks for the PR.
As you noticed, the `joy_teleop_example.yaml` file is only an example of a config file to help one getting started in writing his/her own. It is not necessary to any functionality of the package. Therefore I'd be reluctant to introduce a dependency that is not necessary (e.g. the package did not initially depend on `action_tutorals`).
I'd propose that we keep the change to `config/joy_teleop_example.yaml` but remove this one for merging.
(ping @bmagyar)

Reply:
I agree, thanks for the note! @youtalk , could you please remove the addition of this new dependency?

Reply:
I've removed. Thank you for your comments. I understood.